### PR TITLE
INT-6515 min resources for small instance of nxrm

### PR DESCRIPTION
**Pull request by jflinchbaugh**

### CODE REVIEW

File: charts/nexus-repository-manager/values.yaml [hunk 1]
```diff
@@ -30,18 +32,16 @@ nexus:
       # nexus.ldap.env.java.naming.security.authentication: simple
   # nodeSelector:
   #   cloud.google.com/gke-nodepool: default-pool
-  resources: {}
+  resources:
```
Comment (joedragons) [inline charts/nexus-repository-manager/values.yaml:35]: Is this change right?  Isn't it supposed to follow by definition (whereas before it was empty)?

Comment (jflinchbaugh) [reply charts/nexus-repository-manager/values.yaml:35]: yeah, newer helm is OK with not having that place holder.  it continues to work fine with it gone, and it's one less thing to modify when you choose to uncomment something below.

Comment (joedragons) [reply charts/nexus-repository-manager/values.yaml:35]: Ah nice, thanks for the lesson :)

**Review by joedragons — APPROVED** [review]
Great change, as I think we get semi-regular complaints about underresourced instances.  May even be worth a release note if the Repo team agrees.

**Review by hectorhuol — APPROVED** [review]
+1 LGTM

**Pull request merged**